SConstruct

```diff
@@ -604,11 +604,14 @@ else:
             'target_stdio',
             'target_gcc',
             'target_libuv',
+            'target_linux',
+            'target_bsd',
```

Comment:
With this two lines we enable target_linux and target_bsd on all three platforms: linux, android, macos. We should not do it. We should enable target_linux only on linux and android, and target_bsd only on macos (darwin). Since we already do it below, these two lines can be just removed.